### PR TITLE
[TypeScript] insomnia-xpath

### PR DESCRIPTION
**Pull request by dimitropoulos**

### CODE REVIEW

**Comment by dimitropoulos** [comment]
although making separate branches is much nicer for GitHub/review purposes, only those packages that have no other dependencies from the project will work since they consume each others' types.  For example, insomnia-inso consumes types from insomnia-testing.  If I pull those changes out, it will error because it's expecting types.  For that reason I've pushed a branch [ts/first7](https://github.com/dimitropoulos/insomnia/commits/ts/first7) that contains all of these exact same commits, but stacked.  If you want to run something, your best bet will be to do so from there.  I'll then make sure that everything is always symmetrical between the two.

File: packages/insomnia-xpath/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./tsconfig.build.json",
+  "compilerOptions": {
+    "rootDir": "."
+  },
+  "include": [
+    "src",
+    "jest.config.js",
```
Comment (sonicyeti) [inline packages/insomnia-xpath/tsconfig.json:8]: 🧹

**Review by sonicyeti — CHANGES_REQUESTED** [review]
Looks clean, extra comma on the tsconfig.file.

File: packages/insomnia-xpath/tsconfig.json (hunk 1 above)
Comment (dimitropoulos) [reply packages/insomnia-xpath/tsconfig.json:8]: a decision needs to be made here https://kongstrong.slack.com/archives/GPARXT60G/p1616428970018300

File: packages/insomnia-xpath/package.json [hunk 2]
```diff
@@ -4,10 +4,27 @@
   "author": "Gregory Schier <greg.schier@konghq.com>",
   "description": "Query XML using XPath",
   "license": "MIT",
-  "main": "index.js",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "package.json",
+    "READEME.md"
```
Comment (DMarby) [inline packages/insomnia-xpath/package.json:12]:
```suggestion
    "README.md"
```

**Review by DMarby — APPROVED** [review]

File: packages/insomnia-xpath/tsconfig.json (hunk 1 above)
Comment (dimitropoulos) [reply packages/insomnia-xpath/tsconfig.json:8]: in the name of expediency I went through every tsconfig and made sure none had trailing commas.  turns out there were 3 others that had one somewhere.
Screenshot_20210324_094937

I fixed them all

**Comment by dimitropoulos** [comment]
after feedback (and all fixex therein) and/or approval this PR is closing because this branch is now being incorporated into the dimitropoulos/typescript branch. this isn't final, there is another round of review happening when dimitropoulos/typescript is PR'd into Kong/develop.

**Pull request closed without merging**